data/pigui/game.lua

```diff
@@ -789,24 +795,24 @@ local function displayOnScreenObjects(should_show_label)
 		end
 		-- popup content
 		ui.popup("navtarget" .. v.mainBody.label, function()
-			local size = Vector(16,16)
-			ui.icon(getBodyIcon(v.mainBody.body), size, colors.frame)
-			ui.sameLine()
-			if ui.selectable(v.mainBody.label, v.mainBody.body == navTarget, {}) then
-				player:SetNavTarget(v.mainBody.body)
-			end
-			for k,v in pairs(v.others) do
-				ui.icon(getBodyIcon(v.body), size, colors.frame)
-				ui.sameLine()
-				if ui.selectable(v.label, v.body == navTarget, {}) then
-					player:SetNavTarget(v.body)
-				end
-			end
+							 local size = Vector(16,16)
```

Comment:
that's emacs' automatic indentation, I ran it over the entire file (must have forgotten to do it before)